content/references/xrp-ledger-toml.md

```diff
@@ -280,16 +280,50 @@ Domain verification requires establishing a two-way link between the domain oper
 
     - In that `xrp-ledger.toml` file, provide a `[[VALIDATORS]]` entry with the validator's master public key in the `public_key` field.
 
-2. The validator claims ownership by the domain. The instructions for this step are out of scope for this specification.
+2. The validator claims ownership of the domain: 
 
-    <!-- TODO: Link documentation when it's available. -->
-    <!-- Aside: the old way of doing this was to set the `Domain` field of the
-         XRP Ledger account whose public key matched the validator's public key.
-         Since only the holder of the corresponding private key could send
-         transactions from this address, doing so effectively proved ownership
-         of the validator's private key.
-    -->
+    - Ensure that you have access to the validator-keys.json file that you created when first setting up your validator. If you have lost your keys or the keys have been compromised, please [revoke your keys](https://xrpl.org/run-rippled-as-a-validator.html#revoke-validator-keys) and generate new keys. 
 
+        Note: Recall that your validator-keys.json file should be stored **in a location not on your validator**. 
+
+    - **In a location not on your validator**, build the [validator-keys-too1](https://github.com/ripple/validator-keys-tool).  
+
+    - Run the following command to generate a new validator token that incorporates your domain and update your `xrp-ledger.toml` and `rippled.cfg` files:
+
+        ```
```

Comment:
Our tooling doesn't like code fences in bulleted lists. It'll work if you remove the backticks and indent the code sample another 4 spaces.